### ISSUE TITLE
Cached authorization header

### ISSUE THREAD

**Issue by ferranperello** [a85/POSTMan-Chrome-Extension#1455]
I'm trying to send a call to the server authenticated with Basic Auth but after this call, I want to send another call with another user and POSTMAN send the call with the first user. It seems it is using cache.
I don't find the way to remove the cache to use the other user.
I'm trying to launch automation with jetpacks with different users but the calls are always performed with the first user.

**Comment by abhijitkane** [comment]
@ferranperello Is the Interceptor enabled?
You can check the exact request/response payloads - http://blog.getpostman.com/2015/06/13/debugging-postman-requests/. Is the Authorization header being sent for the second request too?

**Comment by ferranperello** [comment]
@abhijitkane I've tried what you mention and I've could see the following.

I've performed the first call with Basic Authorization and I've could see that the Authorization header is being sent correct: Basic ZnBzYW5kMjFAbnRyLmNvbS5lczoxMjMxMjMxMjM=

After that I've performed the same call but changing the authorization header (another user) and the authorization header that is being sent to the server is correct: Basic YXV0b3NhbjFAbnRyLmNvbS5lczoxMjMxMjMxMjM=
But the response is the same and the server returns the same data that the first call.

I've could see that a session cookie is being stored on the first call and after that, every call is done with that session.

Is there any way to disable cookies on POSTMAN?
